docs/README.md

```diff
@@ -1,18 +1,13 @@
-[![Build Status](https://travis-ci.org/avast/datadog-scala-metrics.svg?branch=master)](https://travis-ci.org/avast/datadog-scala-metrics) [ ![Download](https://api.bintray.com/packages/avast/maven/datadog-scala-metrics/images/download.svg) ](https://bintray.com/avast/maven/datadog-scala-metrics/_latestVersion)
+# Datadog Scala Metrics [![Build Status](https://travis-ci.org/avast/datadog-scala-metrics.svg?branch=master)](https://travis-ci.org/avast/datadog-scala-metrics) [![Download](https://api.bintray.com/packages/avast/maven/datadog-scala-metrics/images/download.svg)](https://bintray.com/avast/maven/datadog-scala-metrics/_latestVersion) <img height="40" src="https://typelevel.org/cats/img/cats-badge-tiny.png" align="right"/>
```

Comment:
Why don't you use markdown syntax for images?

Reply:
it's because of the height and align params that make this look a lot nicer